player.go

```diff
@@ -299,3 +299,12 @@ func (p *Player) setMedia(m *Media) error {
 	C.libvlc_media_player_set_media(p.player, m.media)
 	return getError()
 }
+
+// SetXWindow sets the X window to play on
+func (p *Player) SetXWindow(XID uint32) error {
+	if p.player == nil {
+		return errors.New("A player must be initialized first")
+	}
+	C.libvlc_media_player_set_xwindow(p.player, C.uint(XID))
```

Comment:
Please add an empty line after the `if` statement for consistency with the code base.

Reply:
Done